### PR TITLE
Fixes up supply and drop pods

### PR DESCRIPTION
**Pull request by aidan-bowman**
## About the Pull Request

They're buggy and annoying to use, this fixes them.

## Why It's Good For The Game

Makes it easier to do events and shit.

## Changelog

:cl:
admin: Supply and drop pods improved.
/:cl:

### CODE REVIEW

**Comment by aidan-bowman** [comment]
The tests say that `/area/droppod` is unused even though it's attached to the droppod random map?

File: code/modules/random_map/drop/droppod.dm [hunk 1]
```diff
@@ -19,9 +25,9 @@
 	var/auto_open_doors
 
 	var/placement_explosion_dev =   1
-	var/placement_explosion_heavy = 2
-	var/placement_explosion_light = 6
-	var/placement_explosion_flash = 4
+	var/placement_explosion_heavy = 1
+	var/placement_explosion_light = 3
+	var/placement_explosion_flash = 2
```
Comment (Tsurupeta) [inline code/modules/random_map/drop/droppod.dm:30]: whats the point of having heavy explosion if it has same radius as dev?

Comment (aidan-bowman) [reply code/modules/random_map/drop/droppod.dm:30]: lol, lmao even. dunno how explosion values work so i just reduced them all by half

File: code/modules/random_map/drop/supply.dm [hunk 2]
```diff
@@ -43,41 +43,33 @@
...
-					break
-				chosen_loot_types |= adding_loot_type
+		while(1)
+			var/object = input("Type an atom. Enter an empty string to finish.", "Loot Selection", null)
+
+			if(!object)		// exit
+				break
+
+			var/list/types = typesof(/atom)
+			var/list/matches = new()
```
Comment (ivanmixo) [inline code/modules/random_map/drop/supply.dm]: The hell? 

```suggestion
			var/list/matches = list()
```

File: code/modules/random_map/drop/supply.dm [hunk 3]
```diff
@@ -43,41 +43,33 @@
...
+
+			for(var/path in types)
+				if(findtext("[path]", object))
+					matches += path
+
+			if(matches.len==0)
```
Comment (ivanmixo) [inline code/modules/random_map/drop/supply.dm]:
```suggestion
			if(!length(matches))
```

**Review by ivanmixo — CHANGES_REQUESTED** [review]

**Review by ivanmixo — APPROVED** [review]
Prob okay

**Pull request merged**